pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -440,7 +440,7 @@ const inputSourceJournalTemplate = `{{- define "inputSourceJournalTemplate" -}}
     persistent true
     # NOTE: if this does not end in .json, fluentd will think it
     # is the name of a directory - see fluentd storage_local.rb
-    path '/var/log/journal_pos.json'
+    path '/var/lib/fluentd/pos/journal_pos.json'
```

Comment:
> This looks OK to me with the exception of the framework sleep comment. I'm not well versed in the areas touched, is there anything substantial that still needs to be done?

@alanconway I'm going to look into getting the new SCC declared in the CLO manifest so the OLM manages it for us instead of doing it programmatically.

Reply:
> Define a constant for /var/lib/fluentd/pos, it is mentioned a lot.

@alanconway How strongly do you feel about this? It is mentioned a lot, but all these mentions are in the body of template and adding a template parameter for this seems like too much trouble.